frontend/pages/hosts/details/cards/Queries/_styles.scss

```diff
@@ -1,14 +1,20 @@
 .host-queries-card {
   @include vertical-page-tab-panel-layout;
 
```

Comment:
Slight nit, for whatever reason, the empty state for this page are flushed left instead of center so your centering doesn't match figma's flushed left. cc: @mike-j-thomas we should make empty states super clear in our design guide

<img width="639" height="264" alt="Screenshot 2025-12-29 at 2 55 43 PM" src="https://github.com/user-attachments/assets/80840f7a-52ef-4150-978a-f6ce618bd904" />